sdks/python/apache_beam/examples/inference/pytorch_model_per_key_image_segmentation.py

```diff
@@ -33,7 +33,7 @@
 import torch
 from apache_beam.io.filesystems import FileSystems
 from apache_beam.ml.inference.base import KeyedModelHandler
-from apache_beam.ml.inference.base import KeyMhMapping
+from apache_beam.ml.inference.base import KeyModelMapping
```

Comment:
What abot KeyModelHandlerMapping since it actually maps ModelHandler?

Reply:
When discussed offline, KeyModelMapping was the most popular because it’s somewhere in the middle of understandable, concise, and correct. From a user’s perspective I think reading KeyModelMapping([‘key1’], Mh(…)) is still pretty clear and less verbose than your suggestions.

Reply:
> it’s somewhere in the middle of understandable, concise, and correct.

Great summary :)